components/create/CreateLanding.vue

```diff
@@ -28,7 +28,7 @@
       </div>
 
       <div v-else class="pack">
-        <nuxt-link class="pack-box" to="/create/nft">
+        <div class="pack-box" @click="gotoPathAfterLogin('/create/nft')">
```

Comment:
Why don't you keep `nuxt-link` and use `@click.prevent`?

`<div>` with link is bad semantic

Reply:
>Why don't you keep nuxt-link and use @click.prevent?

It's not allowed
![image](https://github.com/kodadot/nft-gallery/assets/31397967/4506dade-d4f3-4187-9a9d-93ad1661ba33)

In this case, it's `<div>` with a click event handler.

Reply:
Do you mean making such a change as in my screenshot? It does not work after I take a try and cause two duplicate url on the same element.
<img width="716" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/e7d0af31-e192-4ad0-85ab-476a2820f724">

Even though I have tried another solution, it is still the same. https://stackoverflow.com/questions/59885664/nuxt-how-to-prevent-nuxt-link-goes-to-another-page

So, I would keep the original implementation in my PR. If you have any other ideas, feel free to let me know.

Reply:
noted, I'll keep you informed